mail-client/bower/bower-0.9.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit multiprocessing
+
+DESCRIPTION="A curses terminal client for the Notmuch email system"
+HOMEPAGE="https://github.com/wangp/bower"
+SRC_URI="https://github.com/wangp/bower/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+DEPEND="app-crypt/gpgme:=
+	>=dev-lang/mercury-11.07
+	sys-libs/ncurses:0="
```

Comment:
Are you claiming it's not needed at runtime as well?

Reply:
Copy, meet Move. 😕

Reply:
@mgorny On the off-chance that I was not clear enough yesterday: `sys-libs/ncurses:0=` is now listed in both DEPEND and RDEPEND. I moved a line instead of copying it.